README.md

```diff
@@ -59,7 +60,7 @@ A collection of common mistakes made when using [Temporal](https://temporal.io)
 - Not draining signals before completing workflow
 - Depending on `ListWorkflow` API outside of debugging/operational use cases
 - Doing work outside of workflow (eg. before starting, or before sending a signal) -- move into the workflow for more durability
-- Wrapping a queue with a workflow (unless you have a good reason!)
+- [Wrapping a queue with a workflow (unless you have a good reason!)](implementing-a-queue-with-a-workflow.md)
```

Comment:
Heads up this link is broken. But actually could we just remove it since there isn't any content yet?

Also my original intent for this was to talk about scheduling tasks in queueing system via activities, and then waiting for them (in another activity). Point being that this is usually unnecessary since Temporal already provides similar durability guarantees to a queue.

Maybe using Temporal to implement a queuing or pub/sub abstraction should be another entry in the list. 😄

Reply:
Done!